vars/helmPullChart.groovy

```diff
@@ -1,7 +1,8 @@
-void call(String chart, Map opts = [:]) {
+void call(String chart, Map opts = [:], String helmChartVersion = '') {
   Map containers = opts.containers ?: [:]
   String helmContainer = containers.helm ?: ''
   List<String> helmOpts = opts.helmOpts ?: ["--entrypoint=''"]
```

Comment:
You could also add this on line 5 like this and remove the argument from the function signature:

```
String helmChartVersion = opts.helmChartVersion ?: ''
```

Reply:
As I understand the `opts` is comming from the jekinsfile. I.e. 

```
opts = [
  buildAgent: 'jenkins-golang',
  aceFile: 'ace.yaml'
]
```

In ace.yaml we have the chart version that the helm command need. How do we then get the chart version value from ace.yaml file to opts file? Or do we then need to set the chart version in both opts and ace.yaml file?